### PR TITLE
String Sort Desc PR 2

### PR DESCRIPTION
**Pull request by Nick-McGee**
This is my pull request for sorting strings in descending order in Java.
The files contained are:

StringSorterDesc.java: This class contains sortStringsDesc, which uses bubble sort to sort the array in descending order. The comparisons are determined the the java method 'compareToIgnoreCase'. This also means comparisons are case-insensitive.

StringSorterDescTest.java: This class contains one JUnit test to demonstrate that sortStringsDesc is working the way it was intended.

README.md has an added description for StringSorterDesc.java and StringSorterDescTest.java

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@
 
 2. Sorting in Descending Order
     Originally Collections.Sort sorts integer elements in an array by ascending order, by overriding "compare()", SortDes.java makes Collections.Sort sort integer in descending order. Additionally, SortDescTest.java provides unit tests to make sure that testing in descending order works.
+
+3. Sorting Strings in Descending Order
+    To use this function, call StringSorterDesc.sortStringsDesc(String[]). The input is an array of strings, and the output is a new array of strings sorted in descending order. StringSorterDescTest.java is included to make sure the sortStringsDesc functions as intended.
```
Comment (eloiseespel) [inline README.md]: Great explanation! Maybe you could just mention that the strings are sorted using bubble sort.

File: src/StringSorterDescTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+class StringSorterDescTest {
+
+    @org.junit.jupiter.api.Test
+    void sortStringsDesc() {
+        String[] testStrings = {"cow", "bat", "dog", "zebra", "llama"};
+        String[] manuallySorted = {"zebra", "llama", "dog", "cow", "bat"};
+        String[] sortedDescArray = StringSorterDesc.sortStringsDesc(testStrings);
+        assertArrayEquals(manuallySorted, sortedDescArray);
```
Comment (eloiseespel) [inline src/StringSorterDescTest.java:10]: This is a great way to test, you could also use the Assert library included in Junit to minimize overhead but using the Jupiter API works great too!

**Review by yyht6300a — APPROVED** [review]
The READEME.md may be unnecessary because we all directly edit it in master branch.
The code is great!

**Review by Ansper11 — APPROVED** [review]
looks good

**Comment by Ansper11** [comment]
the code is great!

**Pull request merged**